### ISSUE TITLE
Can be used binary-crossentropy to classify 3 classes

### ISSUE THREAD

**Issue by smnfouladi** [boncey/Flickr4Java#354]
hi.can be used binary-crossentropy for loss function in compile to classify 3 classes when lables in categorical format?
one_hot_labels = to_categorical(spectlable, num_classes=3)
for example:class0:[1 .0 .0]
class 1:[0 .1 .0]
and class 2:[0 .0 .1]

and compile is:
model.compile(RMSprop(lr=0.001, rho=0.9), loss="binary_crossentropy", metrics=["accuracy"])
